docs/_includes/hero.html

```diff
@@ -12,9 +12,19 @@
 
       <p class="hero_p sans">
         Not into the command line?
+        Download the compiled source:
         <a class="a-link a-link__icon"
-           href="https://npmcdn.com/capital-framework/dist/">
-          <span class="a-link_text">Download the compiled source</span>
+           href="https://unpkg.com/capital-framework/capital-framework.min.css">
+          <span class="a-link_text">
+            CSS
+          </span>
+          {% include icons/download.svg %}
+        </a>
+        <a class="a-link a-link__icon"
+           href="https://npmcdn.com/capital-framework/capital-framework.min.js">
+          <span class="a-link_text">
+            JS
+          </span>
```

Comment:
Could you close up the white space inside the `span` elements to keep the underline from going under the space between the letters and the icon?

Reply:
Fixed 
<img width="706" alt="Screen Shot 2019-06-21 at 2 09 56 PM" src="https://user-images.githubusercontent.com/704760/59942663-4e9f9b00-942e-11e9-9952-9d830dec3de7.png">